manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-    "spark_version": "2.2.1",
+    "spark_version": "2.3.2",
```

Comment:
I was wondering why this was here in the first place but after merging with master I remembered why. It's because when I tried to build the stub and test on soak the dispatcher failed to come up. I then used the latest release branch as the starting point for these fixes.